src/module/module-collection.js

```diff
@@ -53,6 +53,14 @@ export default class ModuleCollection {
 
     parent.removeChild(key)
   }
+
+  isRegistered (path) {
+    const parent = this.get(path.slice(0, -1))
```

Comment:
yes, this feature is to check existence of module, but maybe not the existence of sub-module.  
It's quite like `mkdir` and `mkdirp`